pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <groupId>com.forgerock.sapi.gateway</groupId>
     <artifactId>secure-api-gateway-parent</artifactId>
-    <version>0.9.0-SNAPSHOT</version>
+    <version>0.9.1-rc.1-SNAPSHOT</version>
```

Comment:
Ah, this does look wrong doesn't it?
I see what you meant now. This should be 0.9.0-rc2-SNAPSHOT, then next release would be 0.9.0-rc1. 

Think we have to accept that we will not do release candidates.

Reply:
Or you will need to fix the workflow to be able to set the next development version. In this case it would be 0.9.0-rc.2-SNAPSHOT

Reply:
Yes, having `MAJOR.MINOR.PATCH-rc.x` the release plugin updates only the `PATCH` to increase the value and then the next releases will be the same release candidate sequence but with different semver.

It means if we release the candidate `1.0.0-rc.1` the next development iteration will be `1.0.1-rc.1-SNAPSHOT` with the consequence the next release will be `1.0.1-rc.1`.

I  agree to build the release without candidate at this moment, and if we have time later we can review the process to build release candidates.

What is your thoughts?  I can close this PR and create the release 0.9.0 to have the next development iteration `0.9.1-SNAPSHOT` until we decide to build the final release for customers `1.0.0`

Reply:
Yep, I think that will be the quickest solution for now

Reply:
Keep it simple, unless you **_have_** to complicate life with rc versions etc.